### PR TITLE
Try bump version

### PR DESCRIPTION
**Pull request by S-Nakamur-a**

### CODE REVIEW

**Review by makotoshimazu — APPROVED** [review]

File: src/process.rs [hunk 1]
```diff
@@ -11,37 +11,108 @@
...
+        println!("interactive.");
         let mut stdin = self.stdin;
+
         std::thread::spawn(move || std::io::copy(&mut std::io::stdin(), &mut stdin).unwrap());
-        self.child.wait()?;
+        let mut stdout = self.stdout_reader;
+
+        std::thread::spawn(move || std::io::copy(&mut stdout, &mut std::io::stdout()).unwrap());
+
+        dbg!(self.child.wait_with_output()?.status);
```
Comment (Hakuyume) [inline src/process.rs:115]: dbg

File: src/process.rs [hunk 2]
```diff
@@ -11,37 +11,108 @@
...
+            result.extend_from_slice(&buf);
+            if result.ends_with(pattern) {
+                return Ok(result);
+            }
+        }
+        Err(io::Error::new(io::ErrorKind::Other, "dame"))
+    }
+
+    pub fn interactive(self) -> io::Result<()> {
+        println!("interactive.");
```
Comment (Hakuyume) [inline src/process.rs:107]: println

File: src/pwn.rs [hunk 3]
```diff
@@ -78,24 +78,40 @@ impl Pwn {
...
+    //         }
+    //         Contents64::Raw(data) => (),
+    //         Contents64::Symbols(_) => todo!(),
+    //         Contents64::RelaSymbols(_) => todo!(),
+    //         Contents64::Dynamics(_) => todo!(),
+    //     }
+
+    //     None
+    // }
+
```
Comment (Hakuyume) [inline src/pwn.rs:114]: これいる？

File: src/process.rs [hunk 4]
```diff
@@ -11,37 +11,108 @@
...
+        let mut result = vec![];
+
+        let mut buf = [0; 1];
+        while self.stdout_reader.read_exact(&mut buf).is_ok() {
+            result.extend_from_slice(&buf);
+            if result.ends_with(pattern) {
+                return Ok(result);
+            }
+        }
+        Err(io::Error::new(io::ErrorKind::Other, "dame"))
```
Comment (makotoshimazu) [inline src/process.rs:103]: "`pattern` not found before reaching to the end."

**Pull request merged**